worker/src/durable.rs

```diff
@@ -194,6 +194,32 @@ impl State {
     pub fn _inner(self) -> DurableObjectState {
         self.inner
     }
+
+    pub fn accept_web_socket(&self, ws: &WebSocket) {
```

Comment:
Noo!!!!! This should have been changed as suggested in https://github.com/cloudflare/workers-rs/pull/434#discussion_r1452829930

Reply:
Sorry mate, must have slipped through when I changed the branches over. Feel free to open a PR to change it.